code/game/machinery/doors/airlock.dm

```diff
@@ -79,6 +98,7 @@ var/list/airlock_overlays = list()
 	var/note_overlay_file = 'icons/obj/doors/airlocks/station/overlays.dmi' //Used for papers and photos pinned to the airlock
 	var/normal_integrity = AIRLOCK_INTEGRITY_N
 	var/prying_so_hard = FALSE
+	var/paintable = TRUE
```

Comment:
It's obvious what it does when you inspect the code deeper but probably not when you're skimming through the airlock definition here.
I suggest adding a short comment here. Something like "whether appearance can be charged with airlock painter"

Reply:
Yeah, will add one